woodwork/table_accessor.py

```diff
@@ -530,6 +531,27 @@ def describe(self, include=None):
         ]
         return pd.DataFrame(results).reindex(index_order)
 
+    def value_counts(self, ascending=False, top_n=10, dropna=False):
+        """Returns a list of dictionaries with counts for the most frequent values in each column (only
+            for columns with `category` as a standard tag).
+
+
+        Args:
+            ascending (bool): Defines whether each list of values should be sorted most frequent
+                to least frequent value (False), or least frequent to most frequent value (True).
+                Defaults to False.
+
+            top_n (int): the number of top values to retrieve. Defaults to 10.
+
+            dropna (bool): determines whether to remove NaN values when finding frequency. Defaults
+                to False.
+
+        Returns:
+            top_list (list(dict)): a list of dictionaries for each categorical column with keys `count`
```

Comment:
Don't need to name the return object here `top_list (list(dict))` -> `list(dict):`